Qt/QtMain.cpp

```diff
@@ -354,7 +355,30 @@ bool MainUI::event(QEvent *e)
         NativeKey(KeyInput(DEVICE_ID_MOUSE, ((QWheelEvent*)e)->delta()<0 ? NKCODE_EXT_MOUSEWHEEL_DOWN : NKCODE_EXT_MOUSEWHEEL_UP, KEY_DOWN));
         break;
     case QEvent::KeyPress:
-        NativeKey(KeyInput(DEVICE_ID_KEYBOARD, KeyMapRawQttoNative.find(((QKeyEvent*)e)->key())->second, KEY_DOWN));
+				{
+					auto qtKeycode = ((QKeyEvent*)e)->key();
+					int nativeKeycode = KeyMapRawQttoNative.find(qtKeycode)->second;
```

Comment:
Hm, another tab/space battleground.

I realize this was here before, but I wonder now: is `KeyMapRawQttoNative` guaranteed to have all keys in it?

-[Unknown]